recipe/meta.yaml

```
@@ -15,21 +23,35 @@ source:
     - 0001-windows-find-blas.patch
 
 build:
-  script: CMAKE_GENERATOR="Ninja" {{ PYTHON }} -m pip install . -vv  # [not win]
+  script:
+    # cuQuantum only supports Volta+
+    # TODO: add cc90 here when we migrate to CUDA 11.8/12.0
+    - export AER_CUDA_ARCH="7.0;7.5;8.0;8.6+PTX"  # [linux64 and cuda_compiler_version != "None"]
```

Maybe this would work

```suggestion
    - export AER_CUDA_ARCH="7.0;7.5;8.0;8.6+PTX"  # [(cuda_compiler_version or "").startswith("11")]
```

---

Addressed in #21.